hooks/stakingDataCalculations.ts

```diff
@@ -0,0 +1,54 @@
+import { FeePoolData } from '@synthetixio/queries';
+import Wei, { wei } from '@synthetixio/wei';
+import { WEEKS_IN_YEAR } from '../constants/date';
+import { StakedSNXResponse } from './useStakedSNX';
+
+export const calculateIsBelowCRatio = (
+  currentCRatio: Wei,
+  targetCRatio: Wei,
+  targetThreshold: Wei
+) => currentCRatio.gt(targetCRatio.mul(wei(1).add(targetThreshold)));
+
+export const calculateWeeklyRewards = (
+  sUSDRate: Wei,
+  SNXRate: Wei,
+  previousFeePeriodData?: FeePoolData
+) => {
+  const feesToDistribute = previousFeePeriodData?.feesToDistribute ?? wei(0);
+  const rewardsToDistribute = previousFeePeriodData?.rewardsToDistribute ?? wei(0);
+
+  return sUSDRate.mul(feesToDistribute).add(SNXRate.mul(rewardsToDistribute));
+};
+export const calculateAPRStaked = (
+  stakedValue: Wei,
+  userDebtBalance: Wei,
+  totalsUSDDebt: Wei,
```

Comment:
We defs need a calculators package :)

Reply:
yea maybe. But I'm not sure any other dapps needs these calculation. So if they're only used in staking there's not too much benefit to have them in a seperate package?